jupyterlab_automl/setup.py

```diff
@@ -55,7 +55,7 @@
     install_requires=[
         "google-cloud-storage>=1.24.1", "jupyterlab~=1.2.0",
         "gcp_jupyterlab_shared",
-        "pip @ https://storage.googleapis.com/google-cloud-aiplatform/libraries/python/0.1.0/google-cloud-aiplatform-0.1.0.tar.gz",
+        "google-cloud-aiplatform @ https://storage.googleapis.com/google-cloud-aiplatform/libraries/python/0.1.0/google-cloud-aiplatform-0.1.0.tar.gz",
```

Comment:
I know Josie was asking me about this before I left on vacation, but were you ever able to confirm that this works properly when you do a fresh installation of the package on an actual CAIP Notebook?

Reply:
Yes. I checked earlier and it does work on a fresh installation on a CAIP Notebook.